docs/streams/index.html

```diff
@@ -269,19 +270,19 @@ <h3 style="margin-top: 5.3rem;">Hello Kafka Streams</h3>
     p
   }
 
-  val builder: StreamsBuilder = new StreamsBuilder()
+  val builder: StreamsBuilder = new StreamsBuilder
   val textLines: KStream[String, String] = builder.stream[String, String]("TextLinesTopic")
   val wordCounts: KTable[String, Long] = textLines
     .flatMapValues(textLine => textLine.toLowerCase.split("\\W+"))
     .groupBy((_, word) => word)
-    .count(Materialized.as("counts-store"))
+    .count(Materialized.as("counts-store").withKeySerde(DefaultSerdes.stringSerde))
```

Comment:
@seglo @debasishg do we still need to enforce `withKeySerde` after https://github.com/apache/kafka/pull/4919/files#diff-09491c933b9023904f73054c5246c893R128 has been merged?